From linux-kernel-owner+greg=40kroah.com-S932148AbWHVUnM@vger.kernel.org Tue Aug 22 13:53:04 2006
Date: Tue, 22 Aug 2006 22:40:15 +0200
From: Jules Villard <jvillard@ens-lyon.fr>
To: trivial@kernel.org
Cc: linux-kernel@vger.kernel.org
Subject: USB: fix typo in drivers/usb/gadget/Kconfig
Message-ID: <20060822204015.GB6817@blatterie>
Content-Disposition: inline

From: Jules Villard <jvillard@ens-lyon.fr>

This tiny patch fixes a typo in drivers/usb/gadget/Kconfig. The typo
is present in 2.6.18-rc4 and in the corresponding -mm tree (and AFAIK,
FYI and FWIW was present in previous kernel versions as well).

From: Jules Villard <jvillard@ens-lyon.fr>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/gadget/Kconfig
+++ gregkh-2.6/drivers/usb/gadget/Kconfig
@@ -26,7 +26,7 @@ config USB_GADGET
 	   you need a low level bus controller driver, and some software
 	   talking to it.  Peripheral controllers are often discrete silicon,
 	   or are integrated with the CPU in a microcontroller.  The more
-	   familiar host side controllers have names like like "EHCI", "OHCI",
+	   familiar host side controllers have names like "EHCI", "OHCI",
 	   or "UHCI", and are usually integrated into southbridges on PC
 	   motherboards.
 
